### PR TITLE
Add release notes for HC commits

### PR DESCRIPTION
**Pull request by velzevur**
Provide short description for user-visible functionalities added by HC Alpha and subsequent commits.

This PR also introduces a release notes line for #3980 and must be merged only after it had been merged.

### CODE REVIEW

File: docs/release-notes/next-hc/HCAlpha.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+* Introduces a new protocol for PoS consensuses. It is ruled by a smart
+  contract.
+* Introduces two smart contracts to govern a PoS consensus
+* Keyblocks are signed by the miner. This is being validated once nodes are
```
Comment (uwiger) [inline docs/release-notes/next-hc/HCAlpha.md:4]:
```suggestion
* Keyblocks are signed by the miner. This is validated once nodes are
```

File: docs/release-notes/next-hc/HCAlpha.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  syncing. Since backwards compatibility is required, the keyblock's seal is
+  reused for this (where the PoW solution is stored), while the excess is
+  padded with zeroes
+* Block rewards pass through the smart contract.
+* A parrent chain connector is added. It consumes the parent chain node's API
```
Comment (uwiger) [inline docs/release-notes/next-hc/HCAlpha.md:9]:
```suggestion
* A parent chain connector is introduced. It consumes the parent chain node's API
```

File: docs/release-notes/next-hc/HCAlpha.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  and will be used to feed the child chain's consensus. An AE2AE connector is
+  fully operational while a AE2BTC one is stubbed. Since this is mostly in RnD
+  state, once the detailes are flattened using the AE2AE, the AE2BTC will be
```
Comment (uwiger) [inline docs/release-notes/next-hc/HCAlpha.md:12]:
```suggestion
  state, once the details are worked out using the AE2AE, the AE2BTC will be
```

File: docs/release-notes/next-hc/HCAlpha.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+  filled in.
+* The consensus smart contract is extended to have the following
+  functionalities:
+  * Anyone can become a validator if they have enough child chain coins.
+    Validators can set themselves for ONLINE or OFFLINE - only ONLINE ones are
```
Comment (uwiger) [inline docs/release-notes/next-hc/HCAlpha.md:17]:
```suggestion
    Validators can set themselves as ONLINE or OFFLINE - only ONLINE ones are
```

File: docs/release-notes/next-hc/HCAlpha.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+    considered while electing a new leader
+  * If you don't have enough - one can delegate their staking power to a
+    validator of their choice, this becoming a delegate
```
Comment (uwiger) [inline docs/release-notes/next-hc/HCAlpha.md:20]:
```suggestion
  * Users who don't have enough to stake can delegate their staking power to a
    validator of their choice, thus becoming a delegate
```

File: docs/release-notes/next-hc/HCAlpha.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+  * Validators have withdraw limits depending on total staking power or
+    percentage of their own stake in their pool
+  * Validators have branding fields in the contract - name, description and
+    avatar url
+  * There is a preconfigurable delay for staking and another one for unstaking
+  * There is a preconfigurable delay before a validator can  become online
```
Comment (uwiger) [inline docs/release-notes/next-hc/HCAlpha.md:26]:
```suggestion
  * There is a preconfigurable delay before a validator counts as online
```

File: docs/release-notes/next-hc/HCAlpha.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+  * Anyone can become a validator if they have enough child chain coins.
+    Validators can set themselves as ONLINE or OFFLINE - only ONLINE ones are
+    considered while electing a new leader
+  * Users who don't have enough to stake can delegate their staking power to a
+    validator of their choice, thus becoming a delegate
+  * Validators have withdraw limits depending on total staking power or
+    percentage of their own stake in their pool
+  * Validators have branding fields in the contract - name, description and
+    avatar url
+  * There is a preconfigurable delay for staking and another one for unstaking
```
Comment (fabiankrol) [inline docs/release-notes/next-hc/HCAlpha.md:25]: delay for unstaking PR: https://github.com/aeternity/aeternity/pull/3987

**Review by fabiankrol — APPROVED** [review]

**Review by uwiger — APPROVED** [review]

**Pull request merged**